### PR TITLE
docs: update README of the xblock

### PR DESCRIPTION
**Pull request by BryanttV**
### Description

This PR updates the README of the Files Manager XBlock.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,151 +1,176 @@
...
-********************
+|status-badge| |license-badge| |ci-badge|
 
-This XBlock comes with a Docker test environment ready to build, based on the xblock-sdk workbench. To build and run it::
+Purpose
+*******
 
-    $ make dev.run
+Files Manager XBlock is a pluggable extension to the Open edX platform that
+allows course creators to add a file manager to manage files and folders, and
```
Comment (mariajgrimaldi) [inline README.rst:10]:
```suggestion
allows course creators to add a file manager to manage files and folders, and
```
what's manage? can we be more specific?

File: README.rst [hunk 2]
```diff
@@ -1,151 +1,176 @@
...
-    // Translators: This comment will appear in the `.po` file.
-    var message = gettext("Custom message.");
+   **NOTE:** The soported files types are the same as the ones supported by the
+   Open edX platform.
 
-See `edx-developer-guide <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/internationalization/i18n.html#javascript-files>`__
-for more information.
+The instructors have an **Unpublished** folder where they can view the files
+into the course assets that have not been categorized yet in the component.
+This folder is not visible to the students. In addition, the instructors can
```
Comment (mariajgrimaldi) [inline README.rst:71]:
```suggestion
This folder is not visible to students. In addition, the instructors can
```

File: README.rst [hunk 3]
```diff
@@ -1,151 +1,176 @@
...
-This cookiecutter template offers multiple make targets which are shortcuts to
-use `edx-i18n-tools <https://github.com/openedx/i18n-tools>`_.
+View from the Learning Management System (LMS)
+**********************************************
 
-After marking strings as translatable we have to create the raw message catalogs.
-These catalogs are created in ``.po`` files. For more information see
-`GNU PO file documentation <https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html>`_.
-These catalogs can be created by running::
+The students can view and download the files and folders that have been added to the
```
Comment (mariajgrimaldi) [inline README.rst:79]:
```suggestion
The students can view and download files and folders that have been added to the
```

File: README.rst [hunk 4]
```diff
@@ -1,151 +1,176 @@
...
+interactive components.
 
-3.1 Add translated strings
----------------------------
+.. _XBlock: https://openedx.org/r/xblock
 
-After creating the raw message catalogs, all translations should be filled out by the translator.
-One or more translators must edit the entries created in the message catalog, i.e. the ``.po`` file(s).
-The format of each entry is as follows::
+You can see the Images Gallery component in action in the XBlock Workbench.
```
Comment (mariajgrimaldi) [inline README.rst:94]:
```suggestion
You can see the Files Manager component in action in the XBlock Workbench.
```

File: README.rst [hunk 5]
```diff
@@ -1,151 +1,176 @@
...
+    git clone git@github.com:eduNEXT/xblock-filesmanager
+    virtualenv venv/
+    source venv/bin/activate
+    cd xblock-filesmanager
+    make upgrade
+    make install
+    make dev.run
 
-To use translations from transifex use the follow Make target to pull translations::
+Once the process is done, you can interact with the Images Gallery XBlock in
```
Comment (mariajgrimaldi) [inline README.rst:107]:
```suggestion
Once the process is done, you can interact with the Files Manager XBlock in
```
You can also fix the rest of the images gallery references

File: README.rst [hunk 6]
```diff
@@ -1,151 +1,176 @@
...
 
-See `edx-developer-guide <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/internationalization/i18n.html#python-source-code>`__
-for more information.
+Using the Files Manager Component
+*********************************
 
-You can also use ``gettext`` to mark strings in javascript::
+1. Go to edit section of the component from Studio.
+2. Create your folders structure by clicking on the **Create folder** button.
+3. Upload files by clicking on the **Upload files** button.
```
Comment (mariajgrimaldi) [inline README.rst:61]: can we specify that the files uploaded by the xblock are also added to the course assets (so they can be viewed from `Content > Files`)

File: README.rst (hunk 2 above)
Comment (BryanttV) [reply README.rst]: Done

File: README.rst (hunk 3 above)
Comment (BryanttV) [reply README.rst]: Done

File: README.rst (hunk 4 above)
Comment (BryanttV) [reply README.rst]: Done

File: README.rst (hunk 5 above)
Comment (BryanttV) [reply README.rst]: Done

File: README.rst (hunk 6 above)
Comment (BryanttV) [reply README.rst]: Done

**Review by Ian2012 — APPROVED** [review]

**Review by mariajgrimaldi — APPROVED** [review]

**Pull request merged**